Formula/mariadb-connector-odbc.rb

```diff
@@ -0,0 +1,25 @@
+class MariadbConnectorOdbc < Formula
+  desc "MariaDB Standardized database driver"
+  homepage "https://downloads.mariadb.org/connector-odbc/"
+  url "http://nyc2.mirrors.digitalocean.com/mariadb//connector-odbc-3.0.2/mariadb-connector-odbc-3.0.2-ga-src.tar.gz"
+  mirror "http://sfo1.mirrors.digitalocean.com/mariadb//connector-odbc-3.0.2/mariadb-connector-odbc-3.0.2-ga-src.tar.gz"
+  sha256 "eba4fbda21ae9d50c94d2cd152f0ec14dde3989522f41ef7d22aa0948882ff93"
+
+  depends_on "cmake" => :build
+  depends_on "mariadb-connector-c"
+  depends_on "openssl"
+  depends_on "unixodbc"
+
+  def install
+    system "cmake", ".", "-DWITH_UNIXODBC=1", "-DMARIADB_FOUND=1",
+            "-DWITH_OPENSSL=1",
+            "-DOPENSSL_INCLUDE_DIR=/usr/local/opt/openssl/include/",
```

Comment:
pop quiz: do you know what is wrong with this line?

Reply:
It's the fact that it's hard-coding a path to OpenSSL rather than using `Formula["openssl"].include` to obtain the proper include folders -- fixing now!